### PR TITLE
[new release] decoders-yojson, decoders-cbor, decoders, decoders-jsonm, decoders-ezjsonm, decoders-sexplib, decoders-bencode and decoders-msgpck (0.5.0)

### PR DESCRIPTION
**Pull request by mattjbray**
Yojson backend for decoders

- Project page: <a href="https://github.com/mattjbray/ocaml-decoders">https://github.com/mattjbray/ocaml-decoders</a>
- Documentation: <a href="https://mattjbray.github.io/ocaml-decoders/">https://mattjbray.github.io/ocaml-decoders/</a>

##### CHANGES:

* Add `Decoders_msgpck` (mattjbray/ocaml-decoders#26, @c-cube)
* Add `let` operators (mattjbray/ocaml-decoders#24, @c-cube)
* Alias `Decode.map` as `<$>` (mattjbray/ocaml-decoders#21, @hamza0867)

### CODE REVIEW

File: packages/decoders-bencode/decoders-bencode.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "A combinator library for \"decoding\" JSON-like values into your own Ocaml types, inspired by Elm's `Json.Decode` and `Json.Encode`."
+maintainer: ["Matt Bray <mattjbray@gmail.com>"]
+authors: ["Simon Cruanes <simon@imandra.ai>"]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders-bencode/decoders-bencode.0.5.0/opam:13]:
```suggestion
  "dune" {>= "2.0"}
```
They should all be 2.0 according to the dune-project file

File: packages/decoders-cbor/decoders-cbor.0.5.0/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "A combinator library for \"decoding\" JSON-like values into your own Ocaml types, inspired by Elm's `Json.Decode` and `Json.Encode`."
+maintainer: ["Matt Bray <mattjbray@gmail.com>"]
+authors: ["Matt Bray <mattjbray@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders-cbor/decoders-cbor.0.5.0/opam:13]:
```suggestion
  "dune" {>= "2.0"}
```

File: packages/decoders-ezjsonm/decoders-ezjsonm.0.5.0/opam [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "A combinator library for \"decoding\" JSON-like values into your own Ocaml types, inspired by Elm's `Json.Decode` and `Json.Encode`."
+maintainer: ["Matt Bray <mattjbray@gmail.com>"]
+authors: ["Matt Bray <mattjbray@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders-ezjsonm/decoders-ezjsonm.0.5.0/opam:13]:
```suggestion
  "dune" {>= "2.0"}
```

File: packages/decoders-jsonm/decoders-jsonm.0.5.0/opam [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  "A combinator library for \"decoding\" JSON-like values into your own Ocaml types, inspired by Elm's `Json.Decode` and `Json.Encode`."
+maintainer: ["Matt Bray <mattjbray@gmail.com>"]
+authors: ["Matt Bray <mattjbray@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders-jsonm/decoders-jsonm.0.5.0/opam:13]:
```suggestion
  "dune" {>= "2.0"}
```

File: packages/decoders-msgpck/decoders-msgpck.0.5.0/opam [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+authors: [
+  "Matt Bray <mattjbray@gmail.com>" "Simon Cruanes <simon@imandra.ai>"
+]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders-msgpck/decoders-msgpck.0.5.0/opam:15]:
```suggestion
  "dune" {>= "2.0"}
```

File: packages/decoders-sexplib/decoders-sexplib.0.5.0/opam [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  "A combinator library for \"decoding\" JSON-like values into your own Ocaml types, inspired by Elm's `Json.Decode` and `Json.Encode`."
+maintainer: ["Matt Bray <mattjbray@gmail.com>"]
+authors: ["Matt Bray <mattjbray@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders-sexplib/decoders-sexplib.0.5.0/opam:13]:
```suggestion
  "dune" {>= "2.0"}
```

File: packages/decoders-yojson/decoders-yojson.0.5.0/opam [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  "A combinator library for \"decoding\" JSON-like values into your own Ocaml types, inspired by Elm's `Json.Decode` and `Json.Encode`."
+maintainer: ["Matt Bray <mattjbray@gmail.com>"]
+authors: ["Matt Bray <mattjbray@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders-yojson/decoders-yojson.0.5.0/opam:13]:
```suggestion
  "dune" {>= "2.0"}
```

File: packages/decoders/decoders.0.5.0/opam [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+  "A combinator library for \"decoding\" JSON-like values into your own Ocaml types, inspired by Elm's `Json.Decode` and `Json.Encode`."
+maintainer: ["Matt Bray <mattjbray@gmail.com>"]
+authors: ["Matt Bray <mattjbray@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/mattjbray/ocaml-decoders"
+doc: "https://mattjbray.github.io/ocaml-decoders/"
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
```
Comment (mseri) [inline packages/decoders/decoders.0.5.0/opam:13]:
```suggestion
  "dune" {>= "2.0"}
```

**Comment by mattjbray** [comment]
👍

**Comment by mseri** [comment]
Good to go now

**Comment by mattjbray** [comment]
Thank you!

**Pull request merged**